packages/hash/frontend/src/pages/type-editor/blockprotocol-ontology-functions-hook.ts

```diff
@@ -0,0 +1,137 @@
+import { EmbedderGraphMessageCallbacks } from "@blockprotocol/graph";
```

Comment:
Should we add a top-level doc on this one as well expressing its purpose and when we're going to delete it

Reply:
Addressed in 68e724d050c21833ff217ae3c6bb53874abeed5b